sklearn/metrics/tests/test_common.py

```diff
@@ -915,23 +921,18 @@ def test_multilabel_representation_invariance():
                         measure,
                         err_msg="%s failed representation invariance between "
                                 "dense and sparse indicator formats." % name)
-
-
-@pytest.mark.parametrize('name', sorted(MULTILABELS_METRICS))
-def test_raise_value_error_multilabel_sequences(name):
```

Comment:
Why are we removing this test? Is it necessary to do so?

Reply:
I thought so, actually only the first sample has to be removed since it is now considered as a valid sequence